x-pack/auditbeat/Jenkinsfile.yml

```diff
@@ -102,7 +97,6 @@ stages:
         stage: packaging
         when:
             branches: false    ## Only on a PR basis for the time being
-            tags: false        ## packaging on branches/tags is already in place with the downstream build.
```

Comment:
Nit: In the future, I think it would be better to split these kind of changes out into a separate PR. I think we mixed two very different concerns in this single PR.

Reply:
Indeed, Let me remove it to help with the review

Reply:
I reverted those changes now